content/articles/pathlib-vs-os-modules-for-file-and-directory-system-operation/index.md

```diff
@@ -0,0 +1,919 @@
+### Pathlib vs OS Modules for File and Directory System Operation
+### Prerequisites
+- Basic knowledge in Python programming language.
+### Contents
+- [Introduction](#introduction)
+- [Os module](#os-module)
+- [Pathlib module](#pathlib-module)
+- [Theoretical difference between Pathlib and Os](#theoretical-difference-between-Pathlib-and-OS)
+- [Practical difference between Os and Pathlib module](#practical-difference-between-OS-and-Pathlib-module)
+- [Querying current path using os and pathlib](#querying-current-path-using-os-and-pathlib)
+- [Navigating to home directory](#navigating-to-home-directory)
+- [Listing directory content](#listing-directory-content)
+- [Creating a directory and files](#creating-a-directory-and-files)
+- [Checking for an existing file or directory](#checking-for-an-existing-file-or-directory)
+- [Renaming a directory or file](#renaming-a-directory-or-file)
+- [Joining path together utilizing Os and Pathlib module](#joining-path-together-utilizing-os-and-pathlib-module)
+- [Querying path to retrieve a final file or directory name](#querying-path-to-retrieve-a-final-file-or-directory-name)
+- [Removing a directory](#removing-a-directory)
+- [Removing a file](#removing-a-file)
+- [Retrieving and converting a file information](#retrieving-and-converting-a-file-information)
+- [Querying a file extension](#querying-a-file-extension)
+- [Wrapping Up](#wrapping-up)
+
+### Introduction
+File System Operation can be characterized as the example or technique that can be
+used to control or construct files and directories and there are two different ways
+of working with File System Operation, either utilizing functional dependency or utilizing
+Object-Oriented. I will practically highlight the different situations to separate between
+the two kinds of modules python provided for system operation. These modules are OS and Pathlib.
```

Comment:
Bring this to the top

Reply:
Avoid using "I"s. Make it third person.

Reply:
The whole paragraph has every sentence in a new line. Combine them together. 

Do this throughout the article

Reply:
Have a look at introductions in other articles to get an idea

Reply:
State why are we learning this? What are the objectives?

Reply:
Okay @srishilesh 
Thank you so much for these illustrative corrections I will follow every step rendered above.